internal/setup/fetch_test.go

```diff
@@ -12,6 +12,8 @@ import (
 // TODO Implement local test server instead of using live repository.
 
 func (s *S) TestFetch(c *C) {
+	c.Skip("TODO chisel-releases need to be updated with public-keys for this test to pass")
```

Comment:
We should totally uncomment this before merging this PR. However, I dont think we can actually support the old Release format. Just as we discussed last week, it is too late because the existing v0.8.0 version cannot parse the new YAML.

Reply:
For our own tracking, this still needs addressing.

Reply:
After the merges to chisel-releases I uncommented the tests and they are passing.